applications/dashboard/controllers/api/SourcemapsApiController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+use Garden\Web\Data;
+
+/**
+ * API Controller to deliver source map files for compiled static resources: js, css.
+ */
+class SourcemapsApiController extends AbstractApiController {
+
+    /**
+     * Get source map.
+     *
+     * @param string $path Relative path to source map file
+     * @return Data
+     */
+    public function get(string $path): Data {
+        $sourceMapsEnabled = Gdn::config()->get(
```

Comment:
Why are we using globals here instead of injecting the config object?

Reply:
I'll update.